docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -531,16 +507,13 @@ spec:
 EOF
 ----
 
-* Adoption of the neutron-ovn-metadata-agent:
-+
-Neutron-ovn-metadata-agent running on the data plane nodes don't require any
-additional actions nor config adjustments to do during the adoption process.
-When OpenStackDataPlaneDeployment and OpenStackDataPlaneNodeSet will be ready,
-neutron-ovn-metadata-agent should be up and running on the data plane nodes.
+[NOTE]
+`Neutron-ovn-metadata-agent` running on the data plane nodes does not require any additional actions or configuration during adoption.
+When the `OpenStackDataPlaneDeployment` and `OpenStackDataPlaneNodeSet` CRs are ready, `neutron-ovn-metadata-agent` is up and running on the data plane nodes.
```

Comment:
(No action required) I don't have access to the scope document linked from the description, so perhaps my comment is misplaced but: I question the need to have a note saying that "nothing should be done". Do we need such no-op in a guide? Perhaps this section should be removed completely?

Reply:
You're right. We don't need the note.